src/chumpackage.cpp

```diff
@@ -254,6 +254,21 @@ void ChumPackage::setInstalledVersion(const QString &v)
     m_installed_version = v;
     emit updated(m_id, PackageInstalledVersionRole);
     emit updated(m_id, PackageInstalledRole);
+    if (type() == PackageApplicationDesktop && installed()) {
+        auto trfl = Daemon::getFiles(m_pkid_installed);
+        connect(trfl, &Transaction::files, this, [this](
```

Comment:
Is there a reason for capturing [this] in the lambda, it appears unused?

Reply:
If I just put `[]` it complains because it tries to access `m_desktopFile` , `updated` and `m_id`. By using `[=]` o `[&]` it also compiles. Since I have a limited knowledge of C++  I would need to read with some attention the documentation to grasp the differences.

Reply:
@piggz, in my perception @jaimeMF's reply shows, that his C++ skills are not that far beyond mine: Apparently he tried things until it was compiling and working, hence asking for proper semantics, plus reasoning for them is futile.

As you do not seem to oppose this implementation to address issue #105 fundamentally, why don't you make a suggestion or MR on top of this with a semantically proper lambda expression.  Because I am afraid without such a move this MR will stall forever.

Reply:
> @piggz, in my perception @jaimeMF's reply shows, that his C++ skills are not that far beyond mine: Apparently he tried things until it was compiling and working, hence asking for proper semantics, plus reasoning for them is futile.
> 
> As you do not seem to oppose this implementation to address issue #105 fundamentally, why don't you make a suggestion or MR on top of this with a semantically proper lambda expression. Because I am afraid without such a move this MR will stall forever.

Indeed, my C++ skills are quite limited, so feel free to suggest any modification to improve it.

Reply:
@piggz, you seem to have had a change (= enhancement) of the proposed code in mind, please specify which.

Reply:
@piggz, I do not want to stall @jaimeMF's effort forever, hence please speak up or I will merge this in April, because it seems to be working fine and it can always still be improved later, because it is just software.

Reply:
In the file chum.cpp there are multiple cases where `this` is captured in the lambdas, I think I used it as a reference. Would it make it clearer if i used `this->` to access the the fields inside the lambda?